components/common/textLink.js

```diff
@@ -14,7 +14,7 @@ const TextLink = ({ href, text, children, external, ...props }) => (
         </ChakraLink>
       </Link>
     ) : (
-      <ChakraLink href={href} target='_blank' rel='noopener noreferrer' className={styles.link} {...props}>
+      <ChakraLink href={href} className={styles.link} {...props}>
```

Comment:
We still need this :( we need to try and figure out how to accessibly open a new tab? Is that a thing?

Reply:
There is, but meeting the visual warning requirement for accessibility is something I'm not sure how we should handle: 
https://www.w3.org/TR/WCAG20-TECHS/G201.html

Screenreader requirement is easy enough to meet. Most links that open in a new tab/window are external links and we could use an icon to indicate as such which is how it's usually done accessibly, but when we're making some internal links open in a new tab/window (are we?), then the icon would be misleading.

Reply:
could we provide an SR only span within the text link that shows when it's an external link?

Reply:
Yeah that's what I'm going to do; we just need to at some point figure out how to handle the visual indicator to make it fully compliant. :)